array_analyzer/transform/point_registration.py

```diff
@@ -246,4 +246,5 @@ def particle_filter(fiducial_coords,
 
     # Generate transformation matrix
     t_matrix = get_translation_matrix(particle)
-    return t_matrix, min_dist
+    reg_dist = min_dist / (fiducial_coords.shape[0] - nbr_outliers)
+    return t_matrix, reg_dist
```

Comment:
Maybe call it something like `regis_error` or `regis_dist`? Took me a while to figure out what the name `reg_dist` means...  updating the docstring would help too :)